src/components/BarChart.js

```diff
@@ -89,6 +95,8 @@ const Rect = React.createClass({
         onClick={this.props.onClick.bind(null, this.props.value, this.props.label)}
         onMouseOut={this._handleMouseOut}
         onMouseOver={this._handleMouseOver.bind(null, this.props.label, this.props.value, this.props.x, y)}
+        rx={3}
+        ry={3}
```

Comment:
It moves the bars down a little to hide the bottom rounded corners. We only want the top of the bars rounded. It is named `shift`. Do you want me to change it to something little more descriptive?